tests/simulator/diagnostics.py

```diff
@@ -22,32 +22,34 @@ def dup(dic):
     dic.update({"diags_fn": lambda model: dump_all_diags(model.populations)})
     return dic
 
+
 @ddt
 class DiagnosticsTest(unittest.TestCase):
 
-    @data(
+    _test_cases = (
       dup({
         "smallest_patch_size": 5,
         "largest_patch_size": 64}),
       dup({
         "smallest_patch_size": 65,
         "largest_patch_size": 65})
     )
-    def test_dump_diags_with_killing_dman_1d(self, input):
 
-        dim = 1
+    def _test_dump_diags_with_killing_dman_nd(self, dim, **input):
+
         for interp in range(1, 4):
             local_out = out + "_" + str(dim) + "_" + str(interp)
             input["diag_options"]["options"]["dir"] = local_out
 
-            self.dman, self.sim, self.hier = create_simulator(1, interp, **input)
+            self.dman, self.sim, self.hier = create_simulator(dim, interp, **input)
             self.dman.dump(timestamp=0, timestep=1)
 
-            for diagInfo in ph.globals.sim.diagnostics:
-                # diagInfo.quantity starts with a / this interferes with os.path.join, hence   [1:]
-                h5_file = os.path.join(local_out, (diagInfo.quantity + ".h5").replace('/', '_')[1:])
-                self.assertTrue(os.path.exists(h5_file))
-                print("hier", hierarchy_from(h5_filename=h5_file))
+            if dim == 1: # TODO REMOVE IF WHEN PHARESEE SUPPORTS 2D!
```

Comment:
need to write an issue for that one not to forget

Reply:
https://github.com/PHAREHUB/PHARE/issues/275